### DISCUSSION TITLE
Make Issue Forms available for Pull Requests

### DISCUSSION THREAD

**Discussion by airtonix** [community/community#31664] [Pull Requests]
simple enough first goal: 1:1 what we get in issue forms for issues.

second goal: allow us to use github action expressions and context to compute resulting pull request form.

**Comment by thibaudcolas** [comment]
Concurring this would be pretty cool! A very basic use case for us would be to have a field that prompts pull request authors to enter the issue number their pull request is about, if any.

And then, similarly to the [official docs for issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms), having a prompt for people to select which browsers a pull request has been tested in.

**Comment by DNin01** [comment]
I think this would help a lot in creating task lists or to require information like what issue is being resolved.

**Comment by k-doering-NOAA** [comment]
This would be really helpful for the teams I work on! It would be a lot easier for the submitter to see what information should be included (and not opt out of parts of the pull request template).

**Comment by migbash** [comment]
This would be amazing!
